packages/ui-new/src/utils/ThemeConstantsUtil.ts

```diff
@@ -0,0 +1,308 @@
+  '20': '80px',
+  '32': '128px',
+  '64': '256px',
+  '128': '512px',
+  round: '9999px'
+}
+
+export const spacing = {
+  '0.5': '2px',
+  '1': '4px',
+  '2': '8px',
+  '3': '12px',
+  '4': '16px',
+  '5': '20px',
+  '6': '24px',
+  '7': '28px',
+  '8': '32px',
+  '9': '36px',
+  '10': '40px',
+  '12': '48px',
+  '14': '56px',
+  '16': '64px',
+  '20': '80px',
+  '32': '128px',
+  '64': '256px'
+}
+
+const fontFamily = {
+  regular: 'KHTeka',
+  mono: 'KHTekaMono'
+}
+
+const fontWeight = {
+  regular: '400',
+  medium: '500'
+}
+
+const textSizes = {
+  h1: '50px',
+  h2: '44px',
+  h3: '38px',
+  h4: '32px',
+  h5: '26px',
+  h6: '20px',
+  large: '16px',
+  medium: '14px',
+  small: '12px'
+}
+
+const typography = {
+  /* Heading 1 */
+  'h1-regular-mono': { lineHeight: '50px', letterSpacing: '-3px' },
+  'h1-regular': { lineHeight: '50px', letterSpacing: '-1px' },
+  'h1-medium': { lineHeight: '50px', letterSpacing: '-0.84px' },
+
+  /* Heading 2 */
+  'h2-regular-mono': { lineHeight: '64px', letterSpacing: '-2.64px' },
+  'h2-regular': { lineHeight: '64px', letterSpacing: '-0.88px' },
+  'h2-medium': { lineHeight: '64px', letterSpacing: '-0.88px' },
+
+  /* Heading 3 */
+  'h3-regular-mono': { lineHeight: '64px', letterSpacing: '-2.28px' },
+  'h3-regular': { lineHeight: '64px', letterSpacing: '-0.76px' },
+  'h3-medium': { lineHeight: '64px', letterSpacing: '-0.76px' },
+
+  /* Heading 4 */
+  'h4-regular-mono': { lineHeight: '32px', letterSpacing: '-1.92px' },
+  'h4-regular': { lineHeight: '32px', letterSpacing: '-0.32px' },
+  'h4-medium': { lineHeight: '32px', letterSpacing: '-0.32px' },
+
+  /* Heading 5 */
+  'h5-regular-mono': { lineHeight: '26px', letterSpacing: '-1.56px' },
+  'h5-regular': { lineHeight: '26px', letterSpacing: '-0.26px' },
+  'h5-medium': { lineHeight: '26px', letterSpacing: '-0.26px' },
+
+  /* Heading 6 */
+  'h6-regular-mono': { lineHeight: '20px', letterSpacing: '-1.2px' },
+  'h6-regular': { lineHeight: '20px', letterSpacing: '-0.6px' },
+  'h6-medium': { lineHeight: '20px', letterSpacing: '-0.6px' },
+
+  /* Large */
+  'lg-regular-mono': { lineHeight: '16px', letterSpacing: '-0.96px' },
+  'lg-regular': { lineHeight: '18px', letterSpacing: '-0.16px' },
+  'lg-medium': { lineHeight: '18px', letterSpacing: '-0.16px' },
+
+  /* Medium */
+  'md-regular-mono': { lineHeight: '14px', letterSpacing: '-0.84px' },
+  'md-regular': { lineHeight: '16px', letterSpacing: '-0.14px' },
+  'md-medium': { lineHeight: '16px', letterSpacing: '-0.14px' },
+
+  /* Small */
+  'sm-regular-mono': { lineHeight: '12px', letterSpacing: '-0.72px' },
+  'sm-regular': { lineHeight: '14px', letterSpacing: '-0.12px' },
+  'sm-medium': { lineHeight: '14px', letterSpacing: '-0.12px' }
+}
+
+export const styles = {
+  colors,
+  fontFamily,
+  fontWeight,
+  textSizes,
+  typography,
+  tokens: {
+    core: tokens.core,
+    /*
+     * We'll use the dark theme as the default theme
+     * since it has same variables as the light theme
```

Comment:
> ...it has same variables as the light theme
You mean the `core` has same values as light?

Reply:
Almost. `core` has separate values from dark / light theme. I put dark theme as default since it has same object values as light theme.

We can later do something like this.

```ts
export default css`
  button {
    color: ${({ theme }) => theme.textPrimary};
  }
`
````